docs/bigquery/reference.rst

```diff
@@ -9,6 +9,7 @@ Client
 
 .. automodule:: google.cloud.bigquery.client
   :members:
+  :inherited-members:
   :show-inheritance:
```

Comment:
Don't we want to drop the `:show-inheritance:` bit, since the base class is private?

Reply:
I think we'd just want to drop it from QueryJob, ExtractJob, etc. I'm not sure how to do that with the current setup.

Reply:
I don't think there is any relevant (i.e., non-public) inheritance to be shown in the `google.cloud.bigquery.job` module.

Reply:
You're right. Dropped from all modules except `client` and `table`.